tests/unit/test_is_close.py

```diff
@@ -44,26 +48,26 @@ def test_010(json_diff: Path) -> None:
 
     alpha_pydict: PyDictResult = {
         "nside-8": {
-            "columns": ["index", "llh", "E_in", "E_tot"],
+            "columns": ["index", "llh", "E_in", "E_tot"] + position_time_columns,
```

Comment:
Using a constant variable for all the columns would make more sense.

Reply:
done